### PR TITLE
fixed footer width issue

### PR DESCRIPTION
**Pull request by sirjan13**
Footer is now full width;

image

### CODE REVIEW

**Comment by sirjan13** [comment]
@championswimmer this has already been merged

**Comment by hereisnaman** [comment]
bro this is not the problem and you are resolving it in a wrong way.

**Comment by omerjerk** [comment]
@sirjan13 Does this still need to be merged? I find the footer on website all fine.

**Comment by sirjan13** [comment]
You can close it :)

File: css/stylesheet.css [hunk 1]
```diff
@@ -1680,9 +1682,10 @@ footer.bg-dark a {
     }
 
     .footer-1{
-        padding-left: 5%!important;
+		padding-left: 5%!important;
         padding-right: 5%!important;
```
Comment (aayusharora) [inline css/stylesheet.css]: Please remove all `!important` tags. This can be done without using this.
What do we need to overwrite here @sirjan13

**Pull request closed without merging**